package.json

```diff
@@ -42,113 +40,106 @@
-    "react-hot-loader": "1.3.1",
+    "react-dom": "^16.3.2",
+    "react-height": "^3.0.0",
     "react-immutable-proptypes": "2.1.0",
     "react-immutable-pure-component": "^1.1.1",
     "react-markdown": "^2.5.0",
     "react-motion": "^0.5.2",
     "react-object-inspector": "0.2.1",
-    "react-redux": "^4.x.x",
-    "react-split-pane": "0.1.70",
-    "redux": "^3.x.x",
-    "redux-immutable": "3.0.8",
+    "react-redux": "^5.0.7",
+    "react-split-pane": "0.1.77",
+    "redux": "^4.0.0",
+    "redux-immutable": "4.0.0",
     "redux-logger": "*",
     "remarkable": "^1.7.1",
-    "reselect": "2.5.3",
+    "reselect": "3.0.1",
     "sanitize-html": "^1.14.1",
     "scroll-to-element": "^2.0.0",
-    "serialize-error": "2.0.0",
-    "shallowequal": "0.2.2",
+    "serialize-error": "2.1.0",
+    "shallowequal": "1.0.2",
     "swagger-client": "^3.7.2",
     "url-parse": "^1.1.8",
-    "whatwg-fetch": "0.11.1",
-    "worker-loader": "^0.7.1",
+    "whatwg-fetch": "2.0.4",
     "xml": "1.0.1",
     "xml-but-prettier": "^1.0.1",
-    "yaml-js": "0.2.0",
-    "zenscroll": "4.0.1"
+    "yaml-js": "0.2.3",
+    "zenscroll": "4.0.2"
   },
   "devDependencies": {
-    "autoprefixer": "7.1.1",
+    "autoprefixer": "8.3.0",
     "babel-core": "^6.23.1",
-    "babel-eslint": "^7.1.1",
+    "babel-eslint": "^8.2.3",
     "babel-loader": "^7.1.0",
     "babel-plugin-module-alias": "^1.6.0",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-es2015": "^6.22.0",
-    "babel-preset-es2015-ie": "^6.6.2",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.23.0",
     "babel-preset-stage-0": "^6.22.0",
     "babel-runtime": "^6.23.0",
     "chromedriver": "^2.30.1",
     "copy-webpack-plugin": "^4.0.1",
-    "css-loader": "0.28.4",
-    "deep-extend": "^0.5.0",
-    "deepmerge": "^1.3.2",
-    "enzyme": "^2.7.1",
+    "css-loader": "0.28.11",
+    "deepmerge": "^2.1.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "^4.1.1",
     "eslint-plugin-import": "^2.6.0",
-    "eslint-plugin-mocha": "^4.11.0",
+    "eslint-plugin-mocha": "^5.0.0",
     "eslint-plugin-react": "^7.1.0",
-    "extract-text-webpack-plugin": "^2.1.2",
-    "file-loader": "0.11.2",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "file-loader": "1.1.11",
     "git-describe": "^4.0.1",
-    "imports-loader": "0.7.1",
-    "json-loader": "0.5.4",
-    "json-server": "^0.11.0",
-    "karma": "^1.7.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-mocha": "^1.3.0",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "2.0.3",
-    "less": "2.7.2",
-    "license-checker": "^11.0.0",
-    "mocha": "^3.4.2",
+    "imports-loader": "0.8.0",
+    "json-loader": "0.5.7",
+    "json-server": "^0.12.2",
+    "less": "3.0.2",
+    "license-checker": "^18.0.0",
+    "mocha": "^5.1.1",
     "nightwatch": "^0.9.16",
     "node-sass": "^4.5.0",
-    "npm-run-all": "4.0.2",
+    "npm-run-all": "^3.1.1",
```

Comment:
You are actually downgrading here...

Reply:
That happened when I reverted from `yarn`. I'll fix.

Reply:
fixed